src/test/java/se/sundsvall/smloader/service/scheduler/DatabaseCleanerSchedulerShedlockTest.java

```diff
@@ -77,7 +78,7 @@ void verifyShedLockForDatabaseCleaner() {
 			.untilAsserted(() -> assertThat(getLockedAt("dbcleaner"))
 				.isCloseTo(LocalDateTime.now(systemUTC()), within(10, ChronoUnit.SECONDS)));
 
-		verify(databaseCleanerService).cleanDatabase(any(OffsetDateTime.class));
+		verify(databaseCleanerService).cleanDatabase(any(OffsetDateTime.class), anyString());
```

Comment:
Use ErrandConstants.MUNICIPALITY_ID instead of anyString()? 
(this is the value used in the DatabaseCleanerScheduler class)